mmv1/products/developerconnect/go_Connection.yaml

```diff
@@ -0,0 +1,200 @@
+    immutable: true
+properties:
+  - name: 'githubConfig'
+    type: NestedObject
+    description: "Configuration for connections to github.com. "
+    min_version: 'beta'
+    properties:
+      - name: 'githubApp'
+        type: String
+        description:
+          "Required. Immutable. The GitHub Application that was installed to
+          the GitHub user or\norganization. \n Possible values:\n GIT_HUB_APP_UNSPECIFIED\nDEVELOPER_CONNECT\nFIREBASE"
+        min_version: 'beta'
+        required: true
+        immutable: true
+      - name: 'authorizerCredential'
+        type: NestedObject
+        description:
+          "Represents an OAuth token of the account that authorized the Connection,\nand
+          associated metadata. "
+        min_version: 'beta'
+        properties:
+          - name: 'oauthTokenSecretVersion'
+            type: String
+            description:
+              "Required. A SecretManager resource containing the OAuth token
+              that authorizes\nthe connection. Format: `projects/*/secrets/*/versions/*`. "
+            min_version: 'beta'
+            required: true
+          - name: 'username'
+            type: String
+            description: "Output only. The username associated with this token. "
+            min_version: 'beta'
+            output: true
+      - name: 'appInstallationId'
+        type: String
+        description: "Optional. GitHub App installation id. "
+        min_version: 'beta'
+      - name: 'installationUri'
+        type: String
+        description:
+          "Output only. The URI to navigate to in order to manage the installation
+          associated\nwith this GitHubConfig. "
+        min_version: 'beta'
+        output: true
+  - name: 'name'
+    type: String
+    description: "Identifier. The resource name of the connection, in the format\n`projects/{project}/locations/{location}/connections/{connection_id}`. "
+    min_version: 'beta'
+    output: true
+  - name: 'createTime'
+    type: String
+    description: "Output only. [Output only] Create timestamp "
+    min_version: 'beta'
+    output: true
+  - name: 'updateTime'
+    type: String
+    description: "Output only. [Output only] Update timestamp "
+    min_version: 'beta'
+    output: true
+  - name: 'deleteTime'
+    type: String
+    description: "Output only. [Output only] Delete timestamp "
+    min_version: 'beta'
+    output: true
+  - name: 'labels'
+    type: KeyValueLabels
+    description: "Optional. Labels as key value pairs "
+    min_version: 'beta'
+  - name: 'installationState'
+    type: NestedObject
+    description:
+      "Describes stage and necessary actions to be taken by the\nuser to
+      complete the installation. Used for GitHub and GitHub Enterprise\nbased connections. "
+    min_version: 'beta'
+    output: true
+    properties:
+      - name: 'stage'
+        type: String
+        description:
+          "Output only. Current step of the installation process. \n Possible
```

Comment:
This line needs to be edited to not have `values:` be the first term in the line: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/developerconnect/Connection.yaml#L143C21-L143C110

then I think this can be regenerated